lib/api/api_test_mock_data.sql

```diff
@@ -0,0 +1,10 @@
+-- Adding single mock item
+INSERT INTO public.items
```

Comment:
Can't stay I understand why you're using `SQL` to seed data ... 🤷‍♂️ 
Instead of adding this to: https://github.com/dwyl/mvp/blob/main/priv/repo/seeds.exs ...

Reply:
I added this as separate because I wanted to have the folder/files associated with **CI only** and make it only relevant for API definition testing only, making it a different "thing" from **integration/unit testing** and **production**.

Reply:
You can easily segment data in `seeds.exs` to only run in a certain context/env. 
That is the accepted way of doing it in `Elixir/Phoenix`. 
I don't _like_ it. But it's how it's done. So unless we drop `Phoenix` and go with something that allows seeding data using raw `SQL` queries out-of-the box, we're going to keep seeding data in `seeds.exs`. 👍

Reply:
Follow-up `tech-debt` task: https://github.com/dwyl/mvp/issues/278 👌